src/components/network/PageNetworks.vue

```diff
@@ -28,7 +29,18 @@ export default {
       return this.networks.filter(network => !network.testnet)
     },
     testNetworks() {
-      return this.networks.filter(network => network.testnet)
+      const networks = this.networks.filter(network => network.testnet)
+      if (config.development) {
+        networks.push({
+          title: "Local Testnet",
```

Comment:
I guess this isn't technically a local testnet since we need to change the graphql url for that.